### PR TITLE
 Chore: Testing integration components

### PR DESCRIPTION
**Pull request by jeremypress**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jeremypress has signed the CLA. Thanks for the pull request!

File: src/components/ContentOpenWith/__tests__/ExecuteForm-test.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        current: {
+            submit: submitStub,
+        },
+    };
+    const onSubmitStub = jest.fn();
+
+    const getWrapper = props => shallow(<ExecuteForm {...props} />);
+
+    describe('componentDidMount', () => {
+        it('should submit the form and call callback on mount', () => {
```
Comment (DanDeMicco) [inline src/components/ContentOpenWith/__tests__/ExecuteForm-test.js]: You should use ```test``` not ```it```

File: src/components/ContentOpenWith/__tests__/IntegrationPortal-test.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        });
+
+        const containerElement = document.querySelector('div');
+        expect(containerElement).toBeTruthy();
+        expect(containerElement.id).toEqual(INTEGRATION_CONTAINER_ID);
+    });
+
+    describe('componentDidMount()', () => {
+        it('should copy styles and append the container to the integration window', () => {
+            const wrapper = getWrapper({
```
Comment (DanDeMicco) [inline src/components/ContentOpenWith/__tests__/IntegrationPortal-test.js]: Can this be moved into a ```beforeEach```? Every test seems to be doing it

Comment (jeremypress) [reply src/components/ContentOpenWith/__tests__/IntegrationPortal-test.js]: the props do change a bit between cases.

File: src/components/ContentOpenWith/__tests__/ExecuteForm-test.js (hunk 1 above)
Comment (jeremypress) [reply src/components/ContentOpenWith/__tests__/ExecuteForm-test.js]: thanks, will address in a follow up PR

**Pull request merged**